templates/people.html

```diff
@@ -0,0 +1,166 @@
+{% extends "layouts/page-with-menu.html" %}
+{% import "macros/assets.html" as assets_macros %}
+
+{% block page_name %}People{% endblock %}
+
+{% block mobile_page_menu %}
+    {{assets_macros::assets_menu(prefix="page-menu", root=section)}}
+{% endblock %}
+
+{% block page_menu %}
+    {{assets_macros::assets_menu(prefix="page-menu", root=section)}}
+{% endblock %}
+
+{% block page_content %}
+
+<svg style="display: none" xmlns="http://www.w3.org/2000/svg" >
```

Comment:
One potential issue with the current "one person per category" approach is that these categories are sets that might intersect. This will be even more true once we add "subject matter expert" info.

Ex: I am a maintainer, a org member, a subject matter expert (rendering, ecs, assets, etc), and a community member. I am also the "project lead" which may or may be communicated on this page as a category.

The current categories are reasonable / not confusing (community members that join the org would just move into the org category, org members that become maintainers would move up). And I do like the hard lines between them. But as soon as we add SMEs, things start to break down I think.

Maybe the current categories are the right approach, but we then add the concept of "labels for SMEs (very similar to area labels on github) and other things like it? Then we could also sort org members by "SME-ness" to ensure that information is prioritized / discoverable. And in the future if we need more "labels" we could add them that way. Maintainers could also have SME tags.

I do think I'd like the fact that I am the project lead to be communicated (as this is a separate role with separate responsibilities). If we create a new "project lead" category above maintainer and move me up, then it would mean I would no longer be perceived as a "maintainer", despite also having that role. So maybe it should be a "label". But then its a little confusing that "maintainer" is the only role that isn't a "label". So maybe all "roles" should be "labels"?

I'd also like this page to be a minimal introduction to "how the bevy project is run". To start, a short description of maintainer, org member, and community member would be nice, which we could render below each category title. I'd like to write the initial descriptions for these, so maybe I'll just add that in a follow up PR.

When SMEs roll out (for those who are confused, you'll hear about these soon), I'd like the SME concept to be communicated here as well.

Maybe we blur the lines for the Bevy Org by making every "role" a "label" and creating some "implicit" categories via newlines/padding?

Ex: `Project Lead`, `Maintainer`, `SME-Rendering`, etc would be labels.

```
## The Bevy Org
[description of the bevy org, including "project lead", "maintainer", and "SME" label descriptions]

[(cart, project-lead-label, maintainer-label, sme-labels)]

[(mockersf, maintainer-label, sme-labels), (alice-i-cecile, maintainer-label, sme-labels), (robswain, maintainer-label, sme-labels)]

[(someone, sme-labels), (someone_else)]


## Community
```

Reply:
"Labels" / the problem they solve also isn't something we need to solve in this PR. Largely just using this PR as a platform to discuss the problem / hopefully reach consensus.

Reply:
Also, it might be better (for space-saving and prominence reasons) to merge the implicit "project lead" and "maintainer sections